### PR TITLE
fix: MYNW-120 Fix Jumbo pricing

### PR DESCRIPTION
**Pull request by loosefer**
Added Jumbo token to the list of coingecko requested prices

https://mnw.atlassian.net/jira/software/projects/MYNW/boards/1?selectedIssue=MYNW-120

image

### CODE REVIEW

**Review by PinkiNice — APPROVED** [review]
Looks good

**Review by Patrick1904 — DISMISSED** [review]

File: packages/frontend/src/redux/slices/tokenFiatValues/index.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ const fetchTokenFiatValues = createAsyncThunk(
     `${SLICE_NAME}/fetchTokenFiatValues`,
     async (_, {dispatch, getState}) => {
         return Promise.allSettled([
-            dispatch(fetchCoinGeckoFiatValues(['near', 'usn'])),
+            dispatch(fetchCoinGeckoFiatValues(['near', 'usn', 'jumbo'])),
```
Comment (roshaans) [inline packages/frontend/src/redux/slices/tokenFiatValues/index.js:26]: I think that the asset on coingecko that you are actually interested in called ['jumbo-exchange'. ](https://www.coingecko.com/en/coins/jumbo-exchange). The price should be closer to .87 cents according to Ref Finance Pool as well as the correct listing on Coingecko listing.

The app is displaying the incorrect pricing. There are two assets listed on coingecko, one is 'jumbo'. it was last updated on June 6th. 

image

**Review by roshaans — CHANGES_REQUESTED** [review]
Please update the asset ID to reflect the correct asset. Otherwise, changes look good.

File: packages/frontend/src/redux/slices/tokenFiatValues/index.js (hunk 1 above)
Comment (loosefer) [reply packages/frontend/src/redux/slices/tokenFiatValues/index.js]: Thank you, fixed!

**Review by roshaans — APPROVED** [review]
LGTM

**Pull request merged**